src/settings/LoanPolicy/components/ViewSections/RequestManagementSection/RequestManagementSection.test.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import {
+  render,
+  screen,
+  within,
+  fireEvent,
+} from '@testing-library/react';
+
+import '../../../../../../test/jest/__mock__';
+
+import RequestManagementSection from './RequestManagementSection';
+
+describe('RequestManagementSection', () => {
+  const requestManagementLabelId = 'ui-circulation.settings.requestManagement.requestManagement';
+  const mockedFunction = jest.fn(() => null);
+  const defaultProps = {
+    isVisible: true,
+    isRecallsOpen: true,
+    isHoldsOpen: true,
+    getPeriodValue: (id) => id,
+    getCheckboxValue: (id) => id,
+    onSectionToggle: mockedFunction,
+  };
+
+  const getById = (id) => within(screen.getByTestId(id));
+
+  const sectionTest = (section, type) => {
+    const labelId = `ui-circulation.settings.requestManagement.${section}`;
+    const valueId = `requestManagement.${type}.${section}`;
+
+    it(`should render "${section}" section correctly`, () => {
+      expect(getById(`${section}TestId`).getByText(labelId)).toBeVisible();
+      expect(getById(`${section}TestId`).getByText(valueId)).toBeVisible();
+    });
+  };
+
+  const accordionTest = (accordionType, testType = true) => {
+    const labelId = `ui-circulation.settings.requestManagement.${accordionType}`;
+
+    it(`should render "${accordionType}" accordion with passed props`, () => {
+      if (testType) {
+        expect(screen.getByTestId(`${accordionType}AccordionTestId`)).toHaveAttribute('open');
+      } else {
+        expect(screen.getByTestId(`${accordionType}AccordionTestId`)).not.toHaveAttribute('open');
+      }
+
+      expect(getById(`${accordionType}AccordionTestId`).getByText(labelId)).toBeVisible();
+      fireEvent.click((screen.getByTestId(`${accordionType}AccordionTestId`)));
+      expect(mockedFunction).toHaveBeenCalledTimes(1);
+    });
+  };
+
+  describe('when accordions is open', () => {
```

Comment:
'when **accordions are** open' or 'when **accordion is** open'

Reply:
done, thanks!